legal-api/tests/unit/resources/v2/test_business.py

```diff
@@ -278,16 +289,18 @@ def test_get_business_with_allowed_filings(session, client, jwt):
         assert rv.json["business"]["allowedFilings"]
 
 
+# TODO: Works with unique identifiers but DB reset fix will resolve the randomly failing tests (ticket# 20121)
 @pytest.mark.parametrize(
-    "test_name, legal_type, identifier, has_missing_business_info, missing_business_info_warning_expected",
+    "test_name, legal_type, owner_legal_type, identifier, owner_identifier, has_missing_business_info,"
+    + "missing_business_info_warning_expected",
     [
-        ("WARNINGS_EXIST_MISSING_DATA", "SP", "FM0000001", True, True),
-        ("WARNINGS_EXIST_MISSING_DATA", "GP", "FM0000002", True, True),
-        ("NO_WARNINGS_EXIST_NO_MISSING_DATA", "SP", "FM0000003", False, False),
-        ("NO_WARNINGS_EXIST_NO_MISSING_DATA", "GP", "FM0000004", False, False),
-        ("NO_WARNINGS_NON_FIRM", "CP", "CP7654321", True, False),
-        ("NO_WARNINGS_NON_FIRM", "BEN", "CP7654321", True, False),
-        ("NO_WARNINGS_NON_FIRM", "BC", "BC7654321", True, False),
+        # ("WARNINGS_EXIST_MISSING_DATA", "SP", "BEN", "FM0000001", "FM0000001", True, True),
+        # ("WARNINGS_EXIST_MISSING_DATA", "GP", None, "FM0000002", None, True, True),
+        ("NO_WARNINGS_EXIST_NO_MISSING_DATA", "SP", "BEN", "FM0000003", "BC0000003", False, False),
+        # ("NO_WARNINGS_EXIST_NO_MISSING_DATA", "GP", None, "FM0000004", None, False, False),
+        # ("NO_WARNINGS_NON_FIRM", "CP", None, "CP7654321", None, True, False),
+        # ("NO_WARNINGS_NON_FIRM", "BEN", None, "CP7654322", None, True, False),
+        # ("NO_WARNINGS_NON_FIRM", "BC", None, "BC7654323", None, True, False),
```

Comment:
can the commented out tests be uncommented?

Reply:
Fixed